### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by raju-b70** [learndevopsonline/learndevopsonline#1844] [Q&A]
![Screenshot (28)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/a1046403-1270-428f-beea-01441771aec6)
![Screenshot (29)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/c379ccf0-319f-4506-88a0-5562d5f95c6e)

**Comment by raju-b70** [comment]
hi i tried with all(first ss) and without all(2 ss)

**Comment by chaithanya1812** [comment]
Hi raju, I am unable to login to the server.
can you paste public-ip of workstation or join the lab session

##  Issue:- Related to inventory file configuration parsed error  
check In command it should be like ansible_password=DevOps321

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/a19f649e-0a40-4f41-a59c-2439ef4fcddf)

**Reply by raju-b70** [reply]
public-ip :3.238.149.184

**Comment by chaithanya1812** [comment]
# I hope your doubt is cleared!

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/6310c37b-8214-4c59-80a8-5ae54912cb6f)

**Reply by raju-b70** [reply]
yes i got it thank you very much mate
